userguide/content/en/docs/adding-content/shortcodes/index.md

```diff
@@ -172,22 +175,70 @@ section. It's meant to be used in combination with the other blocks shortcodes.
 
 ### alert
 
-The **alert** shortcode creates an alert block that can be used to display
-notices or warnings.
+Use the **alert** shortcode to display notices and warnings. The shortcode
+renders a [Bootstrap alert][]. It can be used with Markdown content and contain
```

Comment:
I think having Bootstrap outside the link is slightly easier to parse:

renders a Bootstrap [alert][]

(also need to update the link target)

Reply:
The only reason I found it a little hard to read was because "Bootstrap alert" can read like "an alert relating to Bootstrap" rather than "an alert, as defined by Bootstrap". "Bootstrap alert component"?